### ISSUE TITLE
Custom Layouts - What is the 'mod' value?

### ISSUE THREAD

**Issue by stevejmu** [vsco/keys#4]
In keys/Layout/sampleLayout.keysjson, what does the "mod":"X" parameter refer to?  I've seen different number mod values (0, 2, 8, etc.), but nothing that tells what that mod value refers to.

**Comment by stevejmu** [comment]
After a little digging, I found this in the Constants.h file:

enum MODIFIERS { 
136     kModifierShift = 1, 
137     kModifierControl = 2, 
138     kModifierOption = 4, 
139     kModifierCommand = 8 
140 }; 

So it looks like mod:0 would be just the key itself, and the list of modifier values above refer to the CMD (Win), OPT (Mac), CTRL, and SHIFT keys with their respective values (which I figured as much but didn't know which values referred to which key).

**Comment by parkr** [comment]
@stevejmu You're right, it's a bit confusing. Maybe we should make a `README.md` in the `Layout` directory. Would you mind taking a stab at that?

**Comment by stevejmu** [comment]
I'm new to GitHub - how do I create a Readme.md file in the Layout directory?  Could we just update the Readme.md in the main directory (or both)? Again, I'm not sure how to edit anything on here.

**Comment by parkr** [comment]
No problem! When looking at the files, click "Layouts". You'll see a plus button just above the file view where you can add a new file to that folder. Call it "README.md" and use Markdown to format it, or submit what you'd like to see and I can format it :)

**Comment by hybridvision** [comment]
Hey @parkr, I know this is a very old ticket but people are still using VSCO keys so some improved documentation could be helpful. Are you still taking pull requests for this repository?

I already had a bit of a go at explaining what I found [here](https://github.com/vsco/keys/issues/18#issuecomment-651331843) but I can formalise it for the documentation.

**Comment by parkr** [comment]
Hey @hybridvision! I no longer work at VSCO and can therefore no longer accept PR’s myself. Sorry!

**Comment by hybridvision** [comment]
Sorry @parkr, I should have checked! I appreciate the quick reply though :)

Looking at the list of members, I'm not sure who is best to ask but maybe @bshackelford - are you still taking PRs?

**Comment by bshackelford** [comment]
@hybridvision Yeah, I'm happy to take a look at any PRs and help do what I can to get it merged. Thanks!

(Also, hi @parkr 👋 )
